src/timepicker/types.js

```diff
@@ -35,6 +35,8 @@ export type TimePickerPropsT<T = Date> = {
   nullable?: boolean,
   placeholder?: string,
   size?: SizeT,
+  minTime?: T,
+  maxTime?: T,
```

Comment:
Can you please add this to the typescript types as well? https://github.com/uber/baseweb/blob/master/src/timepicker/index.d.ts